src/2_RTB/PianoDiQualifica/content.typ

```diff
@@ -578,15 +637,123 @@ table(
+A tal fine, il team ha preferito mantenere registrate le considerazioni e le misurazioni effettuate precedentemente tale decisione. 
+
+
+==== Considerazioni Pre-Rimodulazione BAC 
+Possiamo rilevare da il grafico, che la curva delineata dal valore del AEC pur essendo rimasta sempre al di sopra del valore d'accettazione ha, già a partire dal secondo sprint, mostrato un trend decrescente, il team si aspetta che nel corso dei prossimi sprint il valore continui a scendere fino a rientrare nella fascia accettabile. 
+Nell'eventualità il trend dovesse invertirsi il team dovrebbe rimodulare la quantità di lavoro netta associata al concetto di ora produttiva.
+==== Considerazioni Post-Rimodulazione BAC 
+Nonostante il nuovo valore assunto dal BAC a seguito della rimodulazione porta il valore dell'EAC ad avvicinarsi ad esso e a rientrare all'interno della soglia accettabile, rimane comunque più interessante notare come l'andamento continui ad essere decrescente segno che il team sta sfruttando bene le ore produttive a disposizione. 
+
+
+
+=== Budget Variance e Schedule Variance
+
+#figure(
+  image("./assets/valutazione-metriche/BV_SV.png", width: 80%),
+  caption: [
+    Valutazione Budget Variance e Schedule Variance.
+  ],
+)
+
+Il BV indica se alla data corrente si è speso di più o di meno rispetto a quanto inizialmente previsto nel budget; il SV indica se si è in linea, in anticipo o in ritardo rispetto alla schedulazione delle attività di progetto pianificate. 
+Nonostante gli andamenti delle due metriche fosse nel primo periodo in rapido allontanamento dal valore d'accettazione, a partire dal secondo sprint sembrano stabilizzarsi, segno che le azioni correttive adottate stavano iniziando a mostrare gli effetti desiderati.
+Il cambiamento più significativo si è verificato durante il sesto sprint, quando abbiamo osservato un'inversione di tendenza: entrambe le metriche iniziano a riavvicinarsi al valore desiderato.
+Il team si augura che il way of working consolidato nel corso del progetto permetta di recuperare quanto perso nel corso dei primi sprint. 
+Durante il settimo sprint le due metriche hanno mantenuto l'andamento che ci si augurava. 
+
+
+=== Actual Cost e Estimate To Complete
+
+#figure(
+  image("./assets/valutazione-metriche/AC_ETC.png", width: 80%),
+  caption: [
+    Valutazione Actual Cost e Estimate To Complete.
+  ],
+)
+
+L'AC rappresenta il costo effettivo sostenuto fino a un dato momento, mentre l'ETC rappresenta la stima del costo aggiuntivo necessario per completare il progetto; di conseguenza, ci si aspetta che l'AC cresca e che l'ETC diminuisca in modo sostanzialmente lineare, segno che il progetto ha mantenuto un ritmo regolare di avanzamento. Stabilire l'andamento delle due metriche al momento della revisione #glossary("RTB")è alquanto prematuro, dati i pochi data points a disposizione. 
```

Comment:
- spazio dopo "#glossary("RTB")è";
- ricordo di aver scritto che non avevamo molti data points a disposizione all'inizio quando avevamo solo i dati dei primi due sprint ma forse ora si potrebbe trarre qualche conclusione in più (anche stupida) sull'andamento; tipo che naturalmente ci si aspetta che alla fine AC salga fino al BAC e ETC continui a decrescere fino 0 e che si stanno per "incrociare" verso la metà del progetto (RTB dopo 10 settimane nel nostro caso, quindi la metà delle 20 previste). Anche se noi ne abbiamo previste solo 18.

Reply:
okay questo lo lascio aperto cosi domani, quando inserisco, nuovamente le misurazioni me ne ricordo